mod_reforged/hooks/skills/actives/fling_back_skill.nut

```diff
@@ -0,0 +1,7 @@
+::Reforged.HooksMod.hook("scripts/skills/actives/fling_back_skill", function(q) {
+	// We delay the onFollow of this skill by another 100ms to account for faster combat speed mods
```

Comment:
Please edit the comment to be more comprehensive for those who don't know the bug. Something like: `We delay the onFollow of this skill by another 100ms because otherwise the entity doesn't follow through with this function properly when at very high speeds using some faster combat speed mods`